Sources/MapboxMaps/Gestures/GestureHandlers/SingleTapGestureHandler.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+/// `SingleTapGestureHandler` manages a gesture recognizer looking for single tap touch events
+internal final class SingleTapGestureHandler: GestureHandler {
+
+    internal init(gestureRecognizer: UITapGestureRecognizer,
+                  mapboxMap: MapboxMapProtocol,
+                  cameraAnimationsManager: CameraAnimationsManagerProtocol) {
+        gestureRecognizer.numberOfTapsRequired = 1
+        gestureRecognizer.numberOfTouchesRequired = 1
+        super.init(
+            gestureRecognizer: gestureRecognizer,
+            mapboxMap: mapboxMap,
+            cameraAnimationsManager: cameraAnimationsManager)
+        gestureRecognizer.addTarget(self, action: #selector(handleGesture(_:)))
+    }
+
+    @objc private func handleGesture(_ gestureRecognizer: UITapGestureRecognizer) {
+        switch gestureRecognizer.state {
+        case .recognized:
+            cameraAnimationsManager.cancelAnimations()
+            delegate?.gestureBegan(for: .singleTap)
+        default:
+            break
+        }
+    }
+}
```

Comment:
I think we should remove this. The `cancelAnimations` will soon be handled centrally, and limiting the delegate to only deal with gestures that manipulate the camera seems like a reasonable design as well.

Reply:
From discussion: we'll keep this handler for consistency, and will update it to handle the new delegate methods after landing https://github.com/mapbox/mapbox-maps-ios/pull/697